README.md

```diff
@@ -35,6 +35,23 @@ Then, create a Spotify object and call methods:
     for idx, track in enumerate(results['tracks']['items']):
         print(idx, track['name'])
 
+## Contributing
+If you would like to contribute to spotipy follow these steps:
+### Export the needed environment variables
+```bash
+    export SPOTIPY_CLIENT_ID=client_id_here
+    export SPOTIPY_CLIENT_SECRET=client_secret_here
+    export SPOTIPY_CLIENT_USERNAME=client_username_here # This is actually an id not spotify display name
+    export SPOTIPY_REDIRECT_URIa=http://localhost/ # Make url is set in app you created to get your ID and SECRET
```

Comment:
This looks like a typo. Should be `SPOTIPY_REDIRECT_URI` instead of `SPOTIPY_REDIRECT_URIa` I believe.